bool Watering
uint8 Interval
uint8 Duration
uint8 Zones
uint8 ErrorNr
time StartTime
time FinishedTime
